invenio_deposit/filter.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Filter for assets."""
+
+from __future__ import unicode_literals
+
+from flask import current_app
+
+from webassets.filter.cssrewrite.base import PatternRewriter, urltag_re
+
+
+__all__ = (
+    'CSSUrlFixer',
+)
+
+
+# Helper class as a workaround for the strange cleancss URL handling
+class CSSUrlFixer(PatternRewriter):
```

Comment:
I think you should mention this new class in the commit log too, as it can be useful elsewhere...